db/engine.py

```
@@ -1,24 +1,32 @@
+    db_path = Path.cwd() / "health.db"
+    url = f"sqlite:///{db_path}"
+    return create_engine(
+        url,
+        connect_args={"check_same_thread": False},
+        echo=False,
+    )
+
+
+# Create tables using the default engine at import time so interactive usage
+# behaves as expected. Individual sessions may use engines pointing elsewhere.
 from db import models  # noqa: E402 – side-effect import
```

Ruff (0.12.2)

`db.models` imported but unused

Remove unused import: `db.models`

(F401)